TKeazirian.HTTPServer.Tests/ParserTests.cs

```diff
@@ -6,14 +7,45 @@ public class ParserTests
 {
     private const string NewLine = "\r\n";
 
-    [Fact]
-    public void CanParseBody()
+    [Theory]
+    [InlineData($"Hello{NewLine}how{NewLine}are{NewLine}you")]
+    [InlineData("")]
+    public void CanParseBody(string value)
     {
-        string expectedBody = $"Hello{NewLine}how{NewLine}are{NewLine}you";
-        string testRequest = $"HTTP/1.1 200 OK{NewLine}{NewLine}{expectedBody}";
+        object expectedBody = value;
```

Comment:
object and value are not great names.  What ARE these variables representing? Try to use domain language.

Reply:
I guess `object` is a c-sharp thing, for any `object` (aka, it does not have to be a string). So, maybe that doesn't change.

Reply:
I also see that the `value` is not always a `request`, but if we expect it to be it maybe a better name here.

Reply:
`testBody` ?

Reply:
Being specific about types in a strongly typed language can be helpful. I am not sure what the best practice is here for `object` vs. `string` ... more concerned with the names of the variables you control (aka. `value`).  It took me 10min to figure out you were passing in a `body` here.

Reply:
Got it!  Sorry about that :) I am fixing it all now.  I didn't try to change it because that was what it was out of the box but now that I know I'm able to customize it, I'll be more clear going forward!